addons/im_livechat/static/src/new/core/boot_helpers.js

```diff
@@ -50,16 +50,16 @@ export function createRootNode() {
  *
  * @param {ShadowRoot} root
  */
-export async function initializeLivechatContainer(root) {
-    const stylesLink = document.createElement("link");
-    stylesLink.rel = "stylesheet";
-    stylesLink.href = `${serverUrl}/im_livechat/assets_embed.css`;
+export async function initializeEmbed(root) {
```

Comment:
Renamed `initializeEmbed` to `makeShadow`
Also renamed `createRootNode` to `makeRoot`, and test needs to patch `getTarget` (instead of whole `createRootNode`)